assets/css/tailwindCheckout.css

```diff
@@ -29,6 +29,30 @@ header > div {
     @apply max-w-desktop mx-auto;
 }
 
+a,
+a:hover,
+a:focus {
+    @apply text-black outline-black outline-offset-2;
+}
+
+#checkout-app input:active,
+#checkout-app input:focus,
+button:active,
+button:focus {
+    @apply outline-double outline-offset-2 outline-black border-black !important;
+
+    box-shadow: none !important;
```

Comment:
@aleksandraworhacz this line does not belong to anything, could it be a typo?

Reply:
no, it belongs to `.button:focus`. For some reason applying `.shadow-none`  or `.shadow-transparent` didn't overwrite the default blue color when input had `:focus` /`:active` state. That's why I've added CSS rule here.